WORKSPACE

```diff
@@ -102,6 +102,14 @@ load("@com_github_google_benchmark//:bazel/benchmark_deps.bzl", "benchmark_deps"
 
 benchmark_deps()
 
+load("@io_opentelemetry_cpp//bazel:repository.bzl", "opentelemetry_cpp_deps")
```

Comment:
shouldn't this be in grpc_extra_deps?

Reply:
hm, we tried that but it gives a cyclic dependency error.

Reply:
I think that's because the opentelemetry repo calls `grpc_extra_deps()` - https://github.com/open-telemetry/opentelemetry-cpp/blob/main/WORKSPACE#L15

Reply:
Not sure whether opentelemetry `WORKSPACE` file is evaluated when building gRPC targets from gRPC workspace. As long as application developer can use gRPC OpenTelemetry targets without calling additional functions other than grpc_deps` and `grpc_extra_deps`, this might be fine.